package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "@optbot/template",
-  "version": "0.0.2",
-  "description": "template for starting a new package",
-  "main": "lib/start",
+  "name": "@optbot/eqpuller",
+  "version": "0.0.0",
+  "description": "maintain a list of active equities",
+  "main": "lib/norequire",
```

Comment:
Correct, but I think if you're using a module in any fashion, we should be consistent. Either to use the extension or not. I would personally prefer to fully qualify it, as you have done with the change. I've also seen this recommended in several best practices pages.